tests/Aspire.Playground.Tests/QuarantinedTestAttribute.cs

```diff
@@ -53,4 +51,7 @@ public QuarantinedTestAttribute(string? reason = null)
     {
         Reason = reason;
     }
+
+    public IReadOnlyCollection<KeyValuePair<string, string>> GetTraits()
+        => [new KeyValuePair<string, string>("quarantined", "true")];
```

Comment:
There's one more 
https://github.com/dotnet/aspire/blob/3477fc1fec3fbc145883ff309cff89f6a072b16f/tests/Aspire.TestUtilities/QuarantinedTestAttribute.cs#L39-L54

Reply:
Oh, it's been updated in e36881f96b75707a6a006e3704dc1ca8e14f9b59.